_topic_maps/_topic_map_osd.yml

```diff
@@ -204,6 +204,19 @@ Topics:
     - Name: AWS Elastic File Service CSI Driver Operator
       File: osd-persistent-storage-aws-efs-csi
 ---
+Name: Registry
+Dir: registry
+Distros: openshift-dedicated
+Topics:
+- Name: Registry overview
+  File: index
+- Name: Image Registry Operator in OpenShift Container Platform
```

Comment:
@bhardesty  It needs to update OpenShift Container Platform to "Openshift Dedicated"

Reply:
Fixed.